Arara Code: Modify default visibility of SCM views (Repositories, Graph).

Why a patch: View container hideByDefault defaults cannot be modified by settings.
Touches: src/vs/workbench/contrib/scm/browser/scm.contribution.ts

--- a/src/vs/workbench/contrib/scm/browser/scm.contribution.ts
+++ b/src/vs/workbench/contrib/scm/browser/scm.contribution.ts
@@ -94,7 +94,7 @@
 	singleViewPaneContainerTitle: localize('source control repositories', "Source Control Repositories"),
 	ctorDescriptor: new SyncDescriptor(SCMRepositoriesViewPane),
 	canToggleVisibility: true,
-	hideByDefault: true,
+	hideByDefault: false,
 	canMoveView: true,
 	weight: 20,
 	order: 0,
@@ -134,6 +134,7 @@
 	singleViewPaneContainerTitle: localize('source control graph', "Source Control Graph"),
 	ctorDescriptor: new SyncDescriptor(SCMHistoryViewPane),
 	canToggleVisibility: true,
+	hideByDefault: true,
 	canMoveView: true,
 	weight: 40,
 	order: 2,
